src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -122,27 +153,11 @@ def __init__(
 
     def __call__(
         self,
-        text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
         images: ImageInput = None,
-        tokenize_newline_separately: bool = True,
```

Comment:
Shouldn't this be in the text_kwargs defaults too?

Reply:
This wasn't used anywhere so I removed it. Same with `do_thumbnail`, `do_align_long_axis`, `do_rescale`. Do you think we should handle them still for BC even if they had no impact before?

Reply:
ah, ok, no in that case let's remove :)